section1/set6/exercises/09-days-in-month/main.go

```diff
@@ -88,4 +93,16 @@ package main
 // ---------------------------------------------------------
 
 func main() {
+	// declare global variables
+	a := os.Args
+
+	// check if os.Args is empty
+	if len(a) != 2 {
+		fmt.Println("Give me a month name.")
+	}
+	// check correct month names entered
+	if a[1] != "january", "february", "march", "april", "may", "june",
+		"july", "august", "september", "october", "november", "december" {
+		fmt.Println("")
```

Comment:
incorrect print
Also, check for uppercase or mixed case chars
Also account for leap year

Reply:
currently working on 09, not completely finished
will postpone working on this tomorrow